situations/landmark.json

```diff
@@ -27,5 +27,34 @@
             }
         }
     ]
+},
+{
+    "prompt": "the government is coming to take tests on everyone what do you do",
```

Comment:
change to " The government is coming to take tests, what do you do?"

Reply:
alright